target/linux/mvebu/base-files/etc/board.d/02_network

```diff
@@ -39,6 +39,11 @@ armada-388-clearfog-*)
 		ucidef_add_switch "switch0" \
 			"0:lan:5" "1:lan:4" "2:lan:3" "3:lan:2" "4:lan:1" "5u@eth1" "6:lan:6"
 	;;
+armada-385-turris-omnia)
```

Comment:
Same here. One could argue that it currently isn't ordered alphabetically, but let's not make things worse. Ideally, fix the order in a separate commit first.